content/blog/2022/07/psp-migration-script.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Let me help you migrate your PSPs to Kubewarden policies!
+authors:
+- José Guilherme Vanz
+date: 2022-07-07
+---
+
+As announced in past blog posts, Kubewarden have 100% coverage of the deprecated,
```

Comment:
```suggestion
As announced in past blog posts, Kubewarden offers 100% coverage for the deprecated,
```

Reply:
On top of that, I would be nice to make "past blog posts" link to older blog posts about this topic. If we have more than one, we could have "past" link to "post 1", "blog" link to "post 2" and so on so forth.

I don't know if we ever tagged blog posts, we could also have a "PSP" tag that collects all of them